lib/textris/message.rb

```diff
@@ -81,15 +89,23 @@ def parse_from(from)
     end
 
     def parse_from_dual(from)
-      if (matches = from.to_s.match(/(.*)\<(.*)\>\s*$/).to_a).size == 3 &&
-          Phony.plausible?(matches[2])
-        [matches[1].strip, Phony.normalize(matches[2])]
+      matches = from.match(/(.*)\<(.*)\>\s*$/)
+      return if matches.nil?
```

Comment:
checking nil? maybe better to

`return if !matches`

What do you think?

Reply:
@Lackoftactics maybe `return unless matches`?